be/src/storage/persistent_index.cpp

```diff
@@ -2809,48 +2868,96 @@ Status PersistentIndex::abort() {
     return Status::NotSupported("TODO");
 }
 
+uint64_t PersistentIndex::_l1_l2_file_size() const {
+    uint64_t total_l1_l2_file_size = 0;
+    if (_has_l1) {
+        total_l1_l2_file_size += _l1_vec[0]->file_size();
+    }
+    for (int i = 0; i < _l2_vec.size(); i++) {
+        total_l1_l2_file_size += _l2_vec[i]->file_size();
+    }
+    return total_l1_l2_file_size;
+}
+
+bool PersistentIndex::_stop_do_async_compaction() {
```

Comment:
Use the proactive name like `enable_async_compaction`

Reply:
Done